packages/iframe-execution-environment-service/src/IframeExecutionEnvironmentService.ts

```diff
@@ -281,7 +281,7 @@ export class IframeExecutionEnvironmentService
         if (pluginName) {
           this._onError(pluginName, err);
         }
-        commandStream.off('data', errorHandler);
+        commandStream.removeListener('data', errorHandler);
```

Comment:
was getting this error only inside the extension, maybe some weird resolution for the node streams:

![image](https://user-images.githubusercontent.com/364566/135546916-d5929260-26dd-49b1-a351-c5cd6402d5b9.png)

Reply:
Huh. Strange. It's possible that some of the `readable-stream` dependencies we pull in use Node 8 streams, in which `off` doesn't exist, because it wasn't added until Node 10. One day we'll use the same stream libraries everywhere... (or better yet, async iterators _à la_ [gtor](https://github.com/kriskowal/gtor/blob/master/presentation/README.md)).